book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
```

Comment:
FrameworkExtraBundle is not maintained by us. In the core docs, we always use the bundle names as they are in the code: FrameworkBundle, AsseticBundle, MonologBundle, SensioFrameworkExtraBundle, etc. See http://symfony.com/doc/current/cookbook/serializer.html for instance.